QemuPkg/Library/DfciDeviceIdSupportLib/DfciDeviceIdSupportLib.inf

```diff
@@ -0,0 +1,44 @@
+## @file
+# DfciDeviceIdSupportLib.inf
+#
+# NULL Library for DfciDeviceIdSupportLib.

```

Comment:
If this is the Null lib, should we call it DfciDeviceIdSupportLibNull? Or is this comment left over from copying the null instance?

Reply:
Fixed